yaml/oathkeeper/simple/oathkeeper-api.yaml

```diff
@@ -1,51 +0,0 @@
----
```

Comment:
The general idea of this repository to be a *collection* of different ways to solve things. In fact, if you remove this specific file, several published blog posts would fail. Maybe it would make more sense to have e.g. another directory like `yaml/full-stack/postgres/...` where the files you published reside in?

Reply:
Putting it all in directory named yaml doesn't make sense IMHO, as a shell script to generate secrets dynamically included for now and more would come in future.

How about reverting that file back and moving full-stack example related files under a dir named full-stack, just like [docker based examples](https://github.com/ory/examples/tree/master/full-stack)?

Reply:
makes sense!